OrbitClientGgp/ClientGgp.cpp

```diff
@@ -105,6 +106,26 @@ bool ClientGgp::StopCapture() {
   return capture_client_->StopCapture();
 }
 
+bool ClientGgp::SaveCapture() {
+  LOG("Saving capture");
+  const auto& key_to_string_map = string_manager_->GetKeyToStringMap();
+  std::string file_name = options_.capture_file_name;
```

Comment:
This would create a copy of the string, which I don't think you need. You should be able to use std::string_view here instead.

Reply:
I tried doing this but I keep getting compilation erros, should I update _capture_serializer::IncludeOrbitExtensionInFile_ to receive a _string_view_ as well and the _const str_ in _capture_serializer::Save(_ to a _string_view_? I read the documentation and I am having some difficult to understand the difference between str, str_view and const str and in which cases I should use each of them

Reply:
As discussed in person, I was wrong here. You actually need the copy of the string because you are modifying it below.